common/utils/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -151,10 +165,26 @@ trait Logging {
     if (log.isDebugEnabled) log.debug(msg)
   }
 
+  protected def logDebug(entry: LogEntry): Unit = {
```

Comment:
The project doesn't target logDebug/logTrace for now.

Reply:
Done.